docs/getting-started/use-cases/computervision/annotation/cvat/cvat_quick_guide.md

```diff
@@ -28,7 +28,7 @@ You can also bring your own labeling tool as a reproducible template in Onepanel
 
   ![](/img/quickstart-133251.png)
   :::note
-  Some providers have limits on how many volumes you can attach to a node. The default CVAT template in Onepanel requires 6 volumes, so make sure to pick a machine that can support at least that many volumes.
+  Some providers have limits on how many volumes you can attach to a node.
```

Comment:
Should probably use the same language as before but mention 3 volumes instead of 6.